radix-engine/src/kernel/kernel.rs

```diff
@@ -122,12 +124,12 @@ where
             let access_rules = {
                 let mut access_rules = AccessRules::new();
                 access_rules.set_access_rule_and_mutability(
-                    AccessRuleKey::Native(NativeFn::Account(AccountFn::Deposit)),
+                    AccessRuleKey::ScryptoMethod(ACCOUNT_DEPOSIT_IDENT.to_string()),
```

Comment:
If I understand correctly, the only remaining "native" keys are for methods defined by RENode modules, right?

Reply:
RENode modules as well as transient nodes (e.g. Bucket/Proof/etc.). This is handled in the next PR.